### ISSUE TITLE
Issue with square root sign in preview with undo, and "about Mark Text" dialog overlapped with math preview

### ISSUE THREAD

**Issue by kingvagabond** [marktext/marktext#1632]
### Description

There are two issues as I am using marktext.
(1) When you are edting math block and check menu Help -> About Mark Text, the dialog is hidden by math preview block. See attachment.
(2) There is some problem to preview math block when pressing undo. Especially the \sqrt{} sign is not shown. See attachment.
- [ ] Can you reproduce the issue? 
[x]

### Steps to reproduce

1. [First step]
Press Ctrl-Shift-M to enter math block editing. Type following:
\boxed{
\begin{cases}
 \frac{2}{3} \\
 \sqrt{3.4} \\
 \mathbb{Z^+}
 \end{cases}
 }
2. [Second step]
Click Top left menu sign, then Help -> about Mark Text.
3. [and so on...]
Click inside above math block editing, and delete the \\ after \sqrt{3.4}, so the block code like:
\boxed{
\begin{cases}
 \frac{2}{3} \\
 \sqrt{3.4}
 \mathbb{Z^+}
 \end{cases}
 }

Then click outside of the document to hide math block code. Then press Ctrl-Z for once or twice.
**Expected behavior:**

For the first issue, you should see about Mark Text dialog above math preview block.
For the second issue, you should see square root sign still in preview.
**Actual behavior:**

Actually it is behind math preview block.
![image](https://user-images.githubusercontent.com/3144518/68322744-98a9b600-0089-11ea-8108-3ea30f7fd617.png)

For the second issue, the square root sign is not shown up:
![image](https://user-images.githubusercontent.com/3144518/68322825-c131b000-0089-11ea-9d7b-e9c2ec5220f1.png)

**Link to an example: [optional]**

### Versions

- Mark Text: v0.15.1
- Operating system: Windows 10

**Comment by Jocs** [comment]
<img width="896" alt="截屏2019-11-07下午7 01 43" src="https://user-images.githubusercontent.com/9712830/68383694-1a383b80-0191-11ea-890a-24d6c67125de.png">

You need also delete the `\\` after `{3}`?

**Comment by kingvagabond** [comment]
To delete some \\ is just to make some changes and test preview when undo shortcut Ctrl-Z is pressed. 
You can delete all \\ in each line, then keep pressing Ctrl-Z to undo all of delete and see preview.

![image](https://user-images.githubusercontent.com/3144518/68411637-6b740b00-0150-11ea-899e-95b37f964231.png)

**Comment by kingvagabond** [comment]
Please let me know if this is still not fully got. I can upload a video for this.

**Comment by kingvagabond** [comment]
Recorded as animated gif:

![2019-11-07_11-23-29](https://user-images.githubusercontent.com/3144518/68412025-26040d80-0151-11ea-935d-34ccc7d5b22d.gif)

**Comment by kingvagabond** [comment]
Is this clear enough?

**Comment by Jocs** [comment]
Yes, it clear now, can you also show me the correct math formula?

```md
$$
\boxed{
\begin{cases}
\frac{2}{3} 
\sqrt{3.4} 
\mathbb{Z^+}
\end{cases}
}
$$
```

is it the correct formula?

**Comment by kingvagabond** [comment]
This is just a sample of lines to test. Correct format should be:
$$
\boxed{
\begin{cases}
\frac{2}{3} \\
\sqrt{3.4} \\
\mathbb{Z^+} \\
\end{cases}
}
$$

Basically begin{cases} and end{cases} will show the big { sign on left side. Then you can have multiple lines, each ending with \\ to make the { bigger to contain more lines.

![image](https://user-images.githubusercontent.com/3144518/68419534-04f6e900-0160-11ea-9d63-4e5647c628e7.png)
![image](https://user-images.githubusercontent.com/3144518/68419661-4b4c4800-0160-11ea-877b-4ab1654339c0.png)

**Comment by kingvagabond** [comment]
It's better to learn latex expression for 10 mins to understand how users use this. It would benefit a lot and would be enough to cover 99% of the user cases.
